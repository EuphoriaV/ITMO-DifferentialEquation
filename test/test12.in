3.74
(((x + x) * 9.77) - (8.49 * (x * x)))
3.77
5.07