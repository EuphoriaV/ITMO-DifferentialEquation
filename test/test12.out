-8.2E-6